defrag/__init__.py

```diff
@@ -33,5 +34,17 @@
 NO_LOAD = []
 LOAD = []
 
+
+def pretty_log(comment: str, value: Optional[str] = None, warn: bool = False) -> None:
```

Comment:
Why do we need this? Isn't `LOGGER` enough?

Reply:
Have you tried to see the difference?